bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/component.tsx

```diff
@@ -267,43 +266,44 @@ const ChatMessageList: React.FC<ChatListProps> = ({
                       setUserLoadedBackUntilPage(userLoadedBackUntilPage - 1);
                     }
                     }
-                  >
-                    {intl.formatMessage(intlMessages.loadMoreButtonLabel)}
-                  </ButtonLoadMore>
-                ) : null
-            }
-          </span>
-          <div id="contentRef" ref={contentRef}>
-            <ChatPopupContainer />
-            {
-              // @ts-ignore
-              Array.from({ length: pagesToLoad }, (v, k) => k + (firstPageToLoad)).map((page) => {
-                return (
-                  <ChatListPage
-                    key={`page-${page}`}
-                    page={page}
-                    pageSize={PAGE_SIZE}
-                    setLastSender={setLastSender(lastSenderPerPage.current)}
-                    lastSenderPreviousPage={page ? lastSenderPerPage.current.get(page - 1) : undefined}
-                    chatId={chatId}
-                    markMessageAsSeen={markMessageAsSeen}
-                    scrollRef={messageListRef}
-                    lastSeenAt={lastSeenAt}
-                  />
-                )
-              })
-            }
-          </div>
-          <div ref={messagesEndRef} />
-        </MessageList>
-      </MessageListWrapper >,
-      renderUnreadNotification,
-    ]
+                    setUserLoadedBackUntilPage(userLoadedBackUntilPage - 1);
```

Comment:
Why are you calling `setUserLoadedBackUntilPage` twice here, breaking the component behavior